parselglossy/validate.py

```diff
@@ -0,0 +1,195 @@
+import re
+from typing import Any, Dict
+
+
+JsonDict = Dict[str, Any]
+
+
+class InputError(Exception):
+    pass
+
+
+class TemplateError(Exception):
+    pass
+
+
+def type_matches(value: Any, expected_type: str) -> bool:
+    """Checks whether value has the type expected_type.
+
+    If yes, returns True, otherwise False.
+
+    Allowed types T are: 'str', 'int', 'float', 'complex', 'bool',
```

Comment:
This one under a "Notes" block. Probably the types should be in backticks too, so they render properly.